repository/kafka/operator/params.yaml

```diff
@@ -136,7 +136,7 @@ KERBEROS_REALM:
 
 KERBEROS_KDC_HOSTNAME:
   description: "The name or address of a host running a KDC for the realm."
-  default: "kdc-service.kudo-kafka.svc.cluster.local"
```

Comment:
By changing this - I think you have to update the docs too?

Reply:
it's fine as the new default would work when everything is installed under the same namespace.

I just didn't want to have `kudo-kafka` namespace in defaults. But in the documentation, it's ok to use that.